convert.py

```diff
@@ -210,14 +263,67 @@ def convert_text_chunks_to_speech(ebook_chunks):
     print("All chunks converted.")
     return chunk_mp3_filepaths
 
+def merge_wavs_to_mp3(out_filename, path=".", ms_delay=100):
+    combined_mp3 = AudioSegment.empty()
+    path = os.path.abspath(path)
+    print("Joining audio for {}".format(out_filename))
+    for filename in tqdm(sorted(os.listdir(path))):
+        # print("filename: {}, abs: {}, path: {}".format(filename, os.path.abspath(filename), path))
+        if os.path.splitext(filename)[1] == ".wav":
+            # try:
+            combined_mp3 += AudioSegment.from_wav(os.path.join(path, filename))
+            combined_mp3 += AudioSegment.silent(duration=ms_delay)
+            # except Exception as e:
+            #     print("Exception {}, the file was: {}".format(e, filename))
+    out = os.path.abspath(os.path.join(path, out_filename))
+    combined_mp3.export(out, format="mp3")
+    # Clean up
+    for filename in os.listdir(path):
+        if os.path.splitext(filename)[1] == ".wav":
+            os.remove(os.path.join(path, filename))
+
+def offline_text_chunks_to_speech(ebook_chunks):

```

Comment:
Function documentation would be great :)

Reply:
ack